packages/lib/src/Button.wc.svelte

```diff
@@ -12,11 +12,12 @@
     size = 'md',
     part,
     onclick,
-    href
-  }: ButtonProps = $props();
+    href,
+    class: classes
+  }: ButtonProps & { class?: string } = $props();
```

Comment:
Since we decided our components **shouldn't** support:
`<sesamy-login class="extra-tailwind-override-classes-like-so" />`

I decided it's best to not include this in `ButtonProps`. However this ☝️ is nice to have for situations where our components internally renders this Button and want to render it in some slightly different way.

For example this is the button in `PaywallRenderer.svelte`:
`<Button class="w-full mt-2" variant="secondary">Gå vidare</Button>`

There is an argument for adding a property like "`fullWidth: boolean`". But if our customers should be allowed to apply certain styles via props this is what I propose we do.

It's clean and flexible in my opinion.